pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 40
+excerpt: New features, updates, and known limitations
+render: mustache
+model: ../data.yml
+---
+
+# DC/OS Monitoring v.1.3.1
+
+## Updates
+
+* Download all assets from D2iQ servers instead of GitHub.
+
+# DC/OS Monitoring v.1.3.0
+
+## New features
+
+* alertmanager: Add smtp auth secret support.
+* alertmanager: Add a package option for git ref name.
+* Add an option to configure prometheus remote write and remote read.
+
+## Updates
+
+* Updated SDK to v0.57.3.
+
+## Notes
+
+* A `pod replace` operation is required if there are existing deployments of older versions on DC/OS 2.0. See the SDK release notes for details at <https://github.com/mesosphere/dcos-commons/releases/tag/0.57.3>.
```

Comment:
DC/OS 2.0?

Reply:
It's changelog for 1.3.0
https://github.com/mesosphere/dcos-docs-site/blob/staging/pages/mesosphere/dcos/services/dcos-monitoring/1.3.0/release-notes/index.md#notes